src/app/components/home-feed/home-feed.component.scss

```diff
@@ -1,17 +1,21 @@
 main {
-  padding: 80px 0px;
-  display: grid;
-  grid-template-columns: auto auto auto;
+    padding: 80px 50px;
+    width: 600px;
 }
 
-@media screen and (max-width:400px) {
-  main {
-    grid-template-columns: auto auto;
-  }
+.image-size {
+    width: 30px;
+    height: 30px;
+    object-fit: cover;
 }
```

Comment:
Are we using this image-size css class? Is image-container removed or still in the css file?

Reply:
Removed : and no its not there anymore =)